.github/workflows/manage-stale-issues-and-prs.yml

```diff
@@ -0,0 +1,42 @@
+name: "Manage stale issues and PRs"
+on:
+  # Chosen to be just before London wakes up and way past San Francisco's bedtime.
+  schedule:
+    - cron: "0 8 * * 1-5" # This is in UTC.
+  # Do a dry-run (debug-only: true) whenever this workflow itself is changed.
+  pull_request:
+    paths:
+      - .github/workflows/manage-stale-issues-and-prs.yml
+    types:
+      - opened
+      - synchronize
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v6
+        with:
+          close-issue-message: "" # Leave no comment when closing
+          close-pr-message: "" # Leave no comment when closing
+          days-before-issue-stale: 7
+          days-before-pr-stale: 14
+          days-before-close: 7
+          debug-only: ${{ github.event_name == 'pull_request' }} # Dry-run when true.
+          exempt-issue-labels: blocked,do-not-close,feature-gate,security
+          exempt-pr-labels: blocked,do-not-close,feature-gate,security
+          # No actual changes get made in debug-only mode, so we can raise the operations ceiling.
+          operations-per-run: ${{ github.event_name == 'pull_request' && 2000 || 30}}
+          stale-issue-label: stale
+          stale-issue-message: "" # Leave no comment when marking as stale
+          stale-pr-label: stale
+          stale-pr-message: "" # Leave no comment when marking as stale
+          # Time immemorial when in debug-only mode (ie. on pull requests).
+          # `STALEBOT_START_DATE` otherwise.
+          # You can use this as a killswitch by setting `STALEBOT_START_DATE` in the far future.
+          start-date: ${{ github.event_name == 'pull_request' && '1970-01-01T00:00:00Z' || secrets.STALEBOT_START_DATE }} # ISO 8601 or RFC 2822
```

Comment:
This is a strategy for introducing this thing gradually. I will set _myself_ a reminder to back this off slowly, so that we don't end up with every notification, all at once.